src/components/AllCards/AllCards.test.js

```diff
@@ -18,11 +18,12 @@ describe('<AllCards />', () => {
       },
     ],
     handleClick: jest.fn(),
+    handleFetch: jest.fn(),
     filterText: '',
   };
 
   beforeEach(() => {
-    wrapper = shallow(<AllCards {...props} />);
+    wrapper = mount(<AllCards {...props} />);
```

Comment:
Why mount is needed here? It generated this huge snapshot.

Reply:
Uczę się różnych scenariuszy i zapomniałem zmienić, przepraszam za to.

Reply:
No need to be sorry ;) Just was curious why it's done like this.